chingu-coders/Voyage2-Turtles-02
# second attempt to fix CSS Validator bugs

## Purpose
* CSS Validator was returning buggy results.

## Approach/Fixes
* I suspect the problem was that the value of the `textarea` was not being properly formatted to be sent over URI query. I used `encodeURIComponent` to properly format the value.

## Testing
* checkout this PR.
* add some valid and invalid CSS to the validator and check that the error responses are valid
* purposefully edit the proxy URI so that it is invalid. Confirm that an error message shows in output box.
